stdlib/public/Platform/Platform.swift

```diff
@@ -337,7 +346,7 @@ public var SEM_FAILED: UnsafeMutablePointer<sem_t> {
 #if os(OSX) || os(iOS) || os(watchOS) || os(tvOS)
   // The value is ABI.  Value verified to be correct for OS X, iOS, watchOS, tvOS.
   return UnsafeMutablePointer<sem_t>(bitPattern: -1)
-#elseif os(Linux) || os(FreeBSD)
+#elseif os(Linux) || os(FreeBSD) || os(Windows)
   // The value is ABI.  Value verified to be correct on Glibc.
   return UnsafeMutablePointer<sem_t>(bitPattern: 0)
```

Comment:
Did you verify the value?  Could you add a comment about the library and version where you verified it?